### ISSUE TITLE
How to enable setPersistenceEnabled?

### ISSUE THREAD

**Issue by PimDeWitte** [davecoates/rn-firebase-bridge#2]
Hey!

I see you just added the setPersistenceEnabled. I want to add it but I'm pretty new to the project. FYI - I'm one of the contributors on the Unofficial Firebase Unity SDK, and looking to contribute to this repo also, so any help/extra docs/examples you can give me are hugely appreciated so I can get started!

Cheers

**Comment by davecoates** [comment]
Hi there,

I pushed the changes for setPersistenceEnabled but didn't publish a new version to npm - I've just published 0.0.4 which includes this.

Thanks for your interest, I'll try to write something up soon but as a brief overview this project is basically a thin wrapper of the native SDK's for iOS and Android. In terms of API I've tried to match https://firebase.google.com/docs/reference/js with the big change being most functions return promises due to the async nature of communicating over the react native bridge. 

If you have any specific questions feel free to reply and I'll do my best to answer them
